internal/pkg/stream/cloudformation.go

```diff
@@ -104,15 +109,15 @@ func (s *StackStreamer) Fetch() (next time.Time, err error) {
 		// https://github.com/aws/aws-cdk/blob/43f3f09cc561fd32d651b2c327e877ad81c2ddb2/packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts#L230-L234
 		out, err := s.client.DescribeStackEvents(&cloudformation.DescribeStackEventsInput{
 			NextToken: nextToken,
-			StackName: aws.String(s.stackName),
+			StackName: aws.String(s.stackID),
```

Comment:
oh good is it because `stackID` is generally considered a better way to identify a stack?